smart-send-logistics/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "opr/wceu23-smart-send",
+	"description": "The starter template for the Build a 'not at home' shipping extension in the WooCommerce Checkout Block",
```

Comment:
Incorrect leftover from the tutorial

Reply:
Updated with the description from the plugin.